src/Symfony/Component/ObjectMapper/Exception/MapMultipleAggregateException.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\ObjectMapper\Exception;
+
+/**
+ * @experimental
+ *
+ * @author Martin Komischke <martin.komischke@gmail.com>
+ */
+final class MapMultipleAggregateException extends RuntimeException
```

Comment:
Something like `WrappedMappingException`? This naming idea comes from messenger's `WrappedExceptionsInterface`.